rskj-core/src/main/java/co/rsk/rpc/netty/RskWebSocketJsonRpcHandler.java

```diff
@@ -48,25 +52,25 @@
  */
 
 @Sharable
-public class RskJsonRpcHandler
+public class RskWebSocketJsonRpcHandler
         extends SimpleChannelInboundHandler<ByteBufHolder>
         implements RskJsonRpcRequestVisitor {
-    private static final Logger LOGGER = LoggerFactory.getLogger(RskJsonRpcHandler.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(RskWebSocketJsonRpcHandler.class);
 
     private final EthSubscriptionNotificationEmitter emitter;
     private final JsonRpcSerializer serializer;
 
-    public RskJsonRpcHandler(EthSubscriptionNotificationEmitter emitter, JsonRpcSerializer serializer) {
+    public RskWebSocketJsonRpcHandler(EthSubscriptionNotificationEmitter emitter, JsonRpcSerializer serializer) {
         this.emitter = emitter;
         this.serializer = serializer;
     }
 
     @Override
-    protected void channelRead0(ChannelHandlerContext ctx, ByteBufHolder msg) {
-        try {
-            RskJsonRpcRequest request = serializer.deserializeRequest(
-                    new ByteBufInputStream(msg.copy().content())
-            );
+    protected void channelRead0(ChannelHandlerContext ctx, ByteBufHolder msg) throws IOException {
+        ByteBuf content = msg.content().copy();
```

Comment:
I asume this is the same, but previous one was `msg.copy().content()` and this one is `msg.contents().copy()`. Just to avoid unexpected changes.

Reply:
nice catch, I'll change it, as it was back then

Reply:
changed